org.eclipse.xtext.purexbase.tests/src/org/eclipse/xtext/xbase/junit/evaluation/AbstractXbaseEvaluationTest.java

```diff
@@ -2928,13 +2928,12 @@
 	 */
 	@Test @Ignore public void testTryWithResources_catch() throws Exception {
```

Comment:
btw why is here a ignore?

Reply:
Xbase has no syntax for try with resource. It has to be enabled in sublangs for backwards compat reasons